lib/gitlab/configuration.rb

```diff
@@ -32,6 +32,7 @@ def options
     def reset
       self.endpoint       = ENV['GITLAB_API_ENDPOINT']
       self.private_token  = ENV['GITLAB_API_PRIVATE_TOKEN']
+      self.auth_token     = ENV['GITLAB_API_AUTH_TOKEN']
```

Comment:
I think better to have `oauth = false` instead of introducing `auth_token`. So user can set only `private_token` and change `oauth` option if needed.